docker/images/proxysql/suse-compliant/entrypoint/entrypoint.bash

```
@@ -84,10 +84,41 @@ cp -a systemd proxysql-${CURVER}/etc/
 cp -a etc/proxysql.cnf proxysql-${CURVER}/etc/
 cp -a etc/logrotate.d proxysql-${CURVER}/etc/
 cp -a tools/proxysql_galera_checker.sh tools/proxysql_galera_writer.pl proxysql-${CURVER}/usr/share/proxysql/tools
+
+# Plugin .so artefacts (v4.0+ chassis); see rhel-compliant entrypoint
+# for the full rationale.  Gated on the same build flags so v3.x
+# packaging is unchanged.
+if [[ "${PROXYSQL40:-}" == "1" || "${PROXYSQLGENAI:-}" == "1" ]]; then
+    mkdir -p proxysql-${CURVER}/usr/lib/proxysql
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)